### PR TITLE
MSPSDS-1062 Remove pdf case export

### PR DESCRIPTION
**Pull request by samlionheart**
## Description

Removed the link to pdf export on case page as well as the pdf template. In the future we may still want to use pdf export so the pdf gem can remain.
## Checklist:

- [x] Automated checks are passing locally.
### General testing
- [x] Test without javascript
- [x] Test on small screen
### Accessibility testing
- [x] Works keyboard only
- [ ] Tested with one screen reader
- [x] Zoom page to 400% - content still visible
- [x] Disable css - does content make sense and appear in a logical order?
- [ ] Passes automated checker (automated in build or manual)

### CODE REVIEW

File: mspsds-web/app/views/investigations/ts_investigations/has_corrective_action.html.slim [hunk 1]
```diff
@@ -8,7 +8,7 @@
         h1.govuk-heading-xl
           span.govuk-caption-xl Report an unsafe or non-compliant product
           | Has any corrective action been agreed or taken?
-      = render "form_components/govuk_radios", form: form, key: :further_corrective_action,
+      = render "form_components/govuk_radios", form: form, key: :further_has_corrective_action,
```
Comment (broder) [inline mspsds-web/app/views/investigations/ts_investigations/has_corrective_action.html.slim]: Is this change related to this ticket?

File: docker-compose.yml [hunk 2]
```diff
@@ -102,6 +102,7 @@ services:
       - mspsds-worker
     env_file:
       - .env.keycloak.mspsds
+    command: bash -c "rm -f tmp/pids/server.pid && bundle exec rails s -p 3000 -b '0.0.0.0'"
```
Comment (broder) [inline docker-compose.yml]: Should we add this to `docker-entrypoint.sh` for cosmetics and mspsds?

Comment (samlionheart) [reply docker-compose.yml]: This was accidentally popped in this branch, was something to do with docker-sync-stack but ended up being fixed by doing docker-sync-stack clean, as the server.pid was hidden very well somewhere

Comment (samlionheart) [reply docker-compose.yml]: Although this would be helpful to have as it's good for when things don't close down properly
